agent/main.py

```diff
@@ -62,14 +62,20 @@ def main():
     # Configure Signal Handler
     signal.signal(signal.SIGINT, signal_handler)
 
+    first_arg = sys.argv[1]
+    if (first_arg == "start" or first_arg == "stop") and platform.system() != "Windows":
+        del sys.argv[1]
```

Comment:
why this line?

Reply:
Because otherwise the command is parsed by argparser and throws an error